packages/helpwave_localization/lib/l10n/app_de.arb

```diff
@@ -1,4 +1,5 @@
 {
+  "loginSlogan": "Reite auf der Welle!",
```

Comment:
This sounds weird...
Lets remove localization for this slogan.
```suggestion
  "loginSlogan": "Let's ride the wave!",
```
Or hardcode the slogan into the button.